### PR TITLE
[Filesystem] Fixed some docblocks and typos

### PR DESCRIPTION
**Pull request by samnela**
| Q             | A
| ------------- | ---
| Branch?       |  3.4 
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: src/Symfony/Component/Filesystem/Filesystem.php [hunk 1]
```diff
@@ -519,14 +519,14 @@ public function makePathRelative($endPath, $startPath)
...
-     * @param string       $targetDir The target directory
-     * @param \Traversable $iterator  Iterator that filters which files and directories to copy
-     * @param array        $options   An array of boolean options
-     *                                Valid options are:
-     *                                - $options['override'] If true, target files newer than origin files are overwritten (see copy(), defaults to false)
-     *                                - $options['copy_on_windows'] Whether to copy files instead of links on Windows (see symlink(), defaults to false)
-     *                                - $options['delete'] Whether to delete files that are not in the source directory (defaults to false)
+     * @param string            $originDir The origin directory
+     * @param string            $targetDir The target directory
+     * @param \Traversable|null $iterator  Iterator that filters which files and directories to copy
```
Comment (xabbuh) [inline src/Symfony/Component/Filesystem/Filesystem.php]: We should document the behaviour when `null` is passed here.

File: src/Symfony/Component/Filesystem/Filesystem.php [hunk 2]
```diff
@@ -137,8 +137,8 @@ public function exists($files)
      * Sets access and modification time of file.
      *
      * @param string|iterable $files A filename, an array of files, or a \Traversable instance to create
-     * @param int             $time  The touch time as a Unix timestamp
-     * @param int             $atime The access time as a Unix timestamp
+     * @param int|null        $time  The touch time as a Unix timestamp
```
Comment (xabbuh) [inline src/Symfony/Component/Filesystem/Filesystem.php]: We should then add a short description about the behaviour when `null` is passed.

**Review by xabbuh — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Filesystem/Filesystem.php (hunk 2 above)
Comment (samnela) [reply src/Symfony/Component/Filesystem/Filesystem.php]: Added a short description for $atime too

**Review by xabbuh — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @samnela.

**Pull request merged**